### PR TITLE
write introduction to Project Syn

### PR DESCRIPTION
**Pull request by tobru**

### CODE REVIEW

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -2,13 +2,48 @@
...
+
+=== What's in for: Dev
+
+The tooling provided by Project Syn enables the user of the platform to consume all the available services in self-service. By fully automating the day-2 operations (see below), all consumed services are automatically "managed". There is no need to use different configuration management tooling, it's all defined at the same place: Kubernetes objects in a Git repository.
+
+* Automated service deployment with https://crossplane.io/[Crossplane].
+* Backup of data with https://k8up.io[K8up] and https://restic.readthedocs.io[Restic].
+* GitOps with https://argoproj.github.io/argo-cd/[Argo CD].
+* Secrets management with https://www.vaultproject.io/[Vault].
+* Monitoring and alerting with Prometheus, Alertmanager and Signalilo.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/index.adoc]: Please add links like with the tools mentioned above.

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -2,13 +2,48 @@
...
+* Central cluster registry and inventory including GitOps Git repository management provided by https://github.com/projectsyn/lieutenant-api[Lieutenant API], https://github.com/projectsyn/lieutenant-operator[Lieutenant Operator] and xref:steward::index.adoc[Steward].
+* Automated component maintenance with https://github.com/renovatebot/renovate[Renovate] which helps to keep all the applications up-to-date.
+
+pass:[<!-- vale Microsoft.GenderBias = YES -->]
+
+=== High-Level Architecture
+
+The following diagram shows the current high-level architecture (subject to change):
+
+image::syn_highlevel_architecture.png[]
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/index.adoc:33]: Can we have that as an ASCII diagram to have it editable along with the documentation?

**Comment by akosma** [comment]
LGTM! I've added a separate `.vale.ini` file to avoid using the Asciidoc passthrough comments every time the word "Steward" appears

**Review by srueg — APPROVED** [review]

File: docs/modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (tobru) [reply docs/modules/ROOT/pages/index.adoc:33]: We can, but I prefer to do this $later (to keep it lean ATM)

File: docs/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (tobru) [reply docs/modules/ROOT/pages/index.adoc]: Done in da4645d

**Pull request merged**